gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowExecutionResource.java

```diff
@@ -63,6 +63,12 @@ public FlowExecution get(ComplexResourceKey<FlowStatusId, EmptyRecord> key) {
     return this.flowExecutionResourceHandler.getLatestFlowExecution(context, flowId, count, tag, executionStatus);
   }
 
+  @Finder("latestFlowGroupExecutions")
+  public List<FlowExecution> getLatestFlowGroupExecutions(@Context PagingContext context, @QueryParam("flowGroup") String flowGroup,
```

Comment:
Should we rather just add an @Optional @QueryParam to the existing getLatestFlowExecution finder method?

Reply:
I wanted to differentiate them semantically with a separate name for each finder.  the other issue with making the flow name optional on the existing finder is that it is not top-level, but a component of the `flowId` compound param.

Reply:
I see, then should we have changed getLatestFlowExecution to accept two separate params flowGroup and flowName , instead of accepting FlowId ?

Reply:
that change would break b/w compatibility for existing clients.  also, the existing name is misleading as `latestFlowExecution`, if it would return multiple executions across all flows of the group.

plus, after my latest update to this PR, at alex's suggestion, naming of corresponding params has diverged across the two (i.e. `count` and `countPerFlow`).

do you have a particular concern about adding a new finder?